wca-guidelines.md

```diff
@@ -87,7 +87,7 @@ To be more informative, each Guideline is classified using one of the following
 
 - 4b1+) [REMINDER] The WCA Delegate must never re-generate any scramble sequences to replace other ones for the purpose of filtering. For example, it is not permitted to glance at the scramble sequences for a competition and generate the entire set again in order to generate "fairer" scramble sequences.
 - 4b2+) [CLARIFICATION] In general, all official scramble sequences should be kept secret during the competition and published together after the end of the competition. In some cases (e.g. world records), the organization team may wish to release specific scramble sequences sooner after the end of a group.
-- 4b2++) [ADDITION] Competition organizers should ensure that scramblers, scramble sequences, and partially/fully scrambled puzzles are visually isolated from competitors (see [Regulation A2c](regulations:regulation:A2c)). For example, the scramblers may be located behind a wall, or a sufficiently high divider (e.g. a cardboard divider placed around the sides of a table where the scramblers are seated) may be used so that competitors are not able to see puzzles as the scramblers apply scramble sequences.
+- 4b2++) [REMINDER] In the past, visually isolated scrambled puzzles from competitiors have been a recommendation. This is not longer a recommendation but an obligation.
```

Comment:
Not sure about if this is clear enough

Reply:
```suggestion
- 4b2++) [REMINDER] In the past, visually isolated scrambled puzzles from competitors have been a recommendation. This is not longer a recommendation but an requirement.
```

However, I think I'd take this out completely, since it's better to take out Regulations/Guidelines when possible. I don't think many people will be wondering about this, since we're going to have to communicate it to organizers directly anyhow.

Reply:
I agree that really this isn't that necessary. However other important changes (such as puzzle covers) has a similar guideline so I guess it could be useful to remind this